modules/mono/class_db_api_json.cpp

```diff
@@ -236,7 +236,7 @@ void class_db_api_to_json(const String &p_output_file, ClassDB::APIType p_api) {
 	}
 
 	FileAccessRef f = FileAccess::open(p_output_file, FileAccess::WRITE);
-	ERR_FAIL_COND(!f);
+	ERR_FAIL_COND_MSG(!f, "Cannot open file: " + p_output_file + " .");
```

Comment:
I think paths should be surrounded with quotes for cases like this. I used single quotes in error messages places. The reason is the dot (`.`) at the end may be confused as being part of the path. The space between the path and the dot doesn't change that as the space could also be part of the path.

There are more instances of this in the changes.